openmetadata-ui/src/main/resources/ui/src/constants/Tags.constant.ts

```diff
@@ -11,8 +11,33 @@
  *  limitations under the License.
  */
 
+import { Rule } from 'antd/lib/form';
+import i18n from '../utils/i18next/LocalUtil';
+import { NAME_LENGTH_REGEX, TAG_NAME_REGEX } from './regex.constants';
+
 export const DEFAULT_FORM_VALUE = {
   name: '',
   displayName: '',
   description: '',
 };
+
+export const MUI_NAME_FIELD_RULES: Rule[] = [
+  {
+    required: true,
+    message: i18n.t('label.field-required', {
```

Comment:
Remove translation from constnats instead pass key and translate it in component